README.md

```diff
@@ -1,24 +1,38 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Re-Former
 
-Things you may want to cover:
+> This project is for learning how different tags work for erb templates.
 
-* Ruby version
+## Built With
 
-* System dependencies
+- Ruby
+- Rails
+- Active Record
 
-* Configuration
+## How to run
 
-* Database creation
+1. Open Terminal.
+2. Write git clone https://github.com/hamayun-cpu/re-former.git
+3. Go to the root directory.
+4. Write rails server
```

Comment:
- [ ] you forgot to mention that they need to run `bundle install` and `yarn install`